frontend/src/components/HeaderToolbar.vue

```diff
@@ -0,0 +1,41 @@
+<template lang="pug">
+    v-app
```

Comment:
There can only be one `v-app` throughout the application.

Reply:
resolved